suites/tipset/miner_flow.go

```diff
@@ -171,7 +172,8 @@ func TestMinerMissPoStChallengeWindow(t *testing.T, factory state.Factories) {
 	}
 
 	// Epoch advances to the end of the proving window. Send a sing message to trigger the cron actor send
-	td.ExeCtx.Epoch += power_spec.WindowedPostChallengeDuration + miner_spec.ProvingPeriod
+	// XXX: verify this is correct
+	td.ExeCtx.Epoch += miner_spec.WPoStChallengeWindow + 1
```

Comment:
Do you want to test missing a single PoSt deadline, or missing all of them for a whole period? The above will do the latter.

The right epoch depends on the miner's proving period boundary too.

I'm guessing this won't automatically invoke cron. You need to have the power actor's cron callback invoked to trigger miner cleanup. The code might assume that it's invoked for _every_ epoch, no gaps.

Reply:
The invocation of the cron actor happens below on line 177 (after incrementing the epoch here) by sending a message.